elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -23,7 +23,7 @@
         &.expanded {
             @media screen and (min-width: #{pfe-breakpoint(lg)}) {
                 background-color: #{pfe-var(surface--lightest)};
-                border-top-color: #{pfe-var(surface--accent)};
```

Comment:
A border top on the trigger elements doesn't seem like a surface component to me.  I do think it falls under the `ui` category.  How can we get the color we need for the design without sacrificing the meaning of the variable name?

Reply:
+1, this has been switched to UI